src/style.css

```diff
@@ -1 +1,70 @@
-/* Styles go here */
+* {
+  margin: 0;
+  padding: 0;
+}
+
+body {
+  box-sizing: border-box;
+}
```

Comment:
Generally, you mixed it up:
1. that's a bad idea to remove all margins and paddings for all elements
2. `box-sizing` should be placed under the `*` selector